docs/classad-attributes/job-classad-attributes.rst

```diff
@@ -1343,40 +991,26 @@ all attributes.
     +-------+--------------------+
 
 
-:index:`JobPrio<single: JobPrio; ClassAd job attribute>`
-:index:`job ClassAd attribute<single: job ClassAd attribute; JobPrio>`
-
-
-``JobPrio``
+:classad-attribute:`JobPrio`
     Integer priority for this job, set by *condor_submit* or
     *condor_prio*. The default value is 0. The higher the number, the
     greater (better) the priority.
 
-:index:`JobRunCount<single: JobRunCount; ClassAd job attribute>`
-:index:`job ClassAd attribute<single: job ClassAd attribute; JobRunCount>`
-
-``JobRunCount``
+:classad-attribute:`JobRunCount`
     This attribute is retained for backwards compatibility. It may go
     away in the future. It is equivalent to ``NumShadowStarts`` for all
     universes except **scheduler**. For the **scheduler** universe, this
     attribute is equivalent to ``NumJobStarts``.
 
-:index:`JobStartDate<single: JobStartDate; ClassAd job attribute>`
-:index:`job ClassAd attribute<single: job ClassAd attribute; JobStartDate>`
-
-``JobStartDate``
+:classad-attribute:`JobStartDate`
     Time at which the job first began running. Measured in the number of
     seconds since the epoch (00:00:00 UTC, Jan 1, 1970). Due to a long
     standing bug in the 8.6 series and earlier, the job classad that is
     internal to the *condor_startd* and *condor_starter* sets this to
     the time that the job most recently began executing. This bug is
     scheduled to be fixed in the 8.7 series.
 
-:index:`JobStatus<single: JobStatus; ClassAd job attribute>`
-:index:`job ClassAd attribute<single: job ClassAd attribute; JobStatus>`
-:index:`state<single: state; job>`
```

Comment:
Job state is kind of important.  If might be worth leaving this one extra index entry in.